stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/sync/internal/SyncManager.kt

```diff
@@ -110,6 +110,17 @@ internal class SyncManager(
         }
     }
 
+    /**
+     * Store the latest sync state of events. Sync state is updated whenever sync api returns a success response.
+     */
+    internal suspend fun updateSyncStateForEvents() {
+        syncStateFlow.value?.let { syncState ->
+            val newSyncState = syncState.copy(lastSyncedAt = Date())
```

Comment:
What if the event arrives between making an API call and updating this value? 🤔 
Maybe we should update it with the most recent `event.createdAt` received from the API call?
WDYT?